pkgs/build-support/node/build-npm-package/hooks/npm-config-hook.sh

```diff
@@ -0,0 +1,76 @@
+npmSetupHook() {
```

Comment:
`npmSetupHook` seems like a bit of a generic name, also doesn't match the filename.
```suggestion
npmConfigureHook() {
```